test/stress/large_microbench.c

```diff
@@ -0,0 +1,35 @@
+#include "test/jemalloc_test.h"
+#include "test/bench.h"
+
+static void
+large_mallocx_free(void) {
+	void *p = mallocx(128 * 1024, 0);
```

Comment:
nit: use the smallest large size macro to be more general?

Reply:
Will do.